pkg/apply/task/inv_set_task.go

```diff
@@ -127,3 +130,12 @@ func (i *InvSetTask) Cancel(_ *taskrunner.TaskContext) {}
 
 // StatusUpdate is not supported by the InvSetTask.
 func (i *InvSetTask) StatusUpdate(_ *taskrunner.TaskContext, _ object.ObjMetadata) {}
+
+// getApplyReoncileStatus captures the apply status and reconcile status
+// for each individual resource in the inventory list.
+// The apply/reconcile status is then passed
+// to the inventory client and stored in the inventory object
+// through the Inventory interface.
+func (i InvSetTask) getApplyReoncileStatus(taskContext *taskrunner.TaskContext) []inventory.ObjectStatus {
```

Comment:
I think you can just inline this one-line method.

Reply:
Done